### DISCUSSION TITLE
[5.5.50] Approaches for retaining existing sessions following security update

### DISCUSSION THREAD

**Discussion by Pedromcn** [laravel/framework#34011] [General] [answered]
- Laravel version: 5.5.50
- PHP Version: 7.2

### Description
We're in the process of updating the `illuminate/cookie` library to apply the recent security fix on one of our applications. Reading through the notes & docs, it is highlighted in multiple occasions (and confirmed by our initial tests) that upon rolling out this update to Production, all of our end users will be logged out.
As we maintain an application with thousands of concurrent customers, this will be quite costly from a monetary and customer experience perspective. As such, we would very much like to find ways to prevent this.

### Question
Do you have any suggestions / approaches to allow us to rollout this update while keeping the customers logged-in? Do you have any viable alternatives?

**Comment by driesvints** [comment, marked as answer]
Use `5.5.49` in your composer.json
